win/project/sockperf.vcxproj

```diff
@@ -85,7 +85,7 @@
       </PrecompiledHeader>
       <WarningLevel>Level3</WarningLevel>
       <Optimization>Disabled</Optimization>
-      <PreprocessorDefinitions>WIN32;_DEBUG;_CONSOLE;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <PreprocessorDefinitions>__windows__;WIN32;_DEBUG;_CONSOLE;%(PreprocessorDefinitions)</PreprocessorDefinitions>

```

Comment:
AFAIK, MSVC compiler always defines `WIN32` for both 32- and 64-bit Windows platform so I see no reason to define one more macro.

Reply:
It was agreed on the call with @igor-ivanov  that WIN32 is very confusing, although it supports both 32 and 64 bit. 
The idea is to be aligned with `__linux__` macro.